### ISSUE TITLE
oidc-token --id-token operation forbidden on Windows 11

### ISSUE THREAD

**Issue by tabramczyk** [indigo-dc/oidc-agent#556]
Hello,

I can't obtain id-token on Windows 11. 
I'm using oidc-agent 5.0.1.

How to reproduce:
1. kill all oidc-agent processes
2. open Windows cmd and go to oidc-agent installation directory
3. oidc-agent.exe
4. oidc-add.exe <NAME>
5. oidc-token.exe loki-auth --id-token
6. you will see: operation forbidden

We have this problem on several machines with Windows.

**Comment by zachmann** [comment]
I tried to reproduce the issue. And in principle it works for me. 
However, usage of the `--id-token` option (by default) always has to be confirmed manually by the user.
This means, that `oidc-prompt` will open a GUI window and ask the user to confirm the usage. If the confirmation is declined (or the user cannot be prompted - but this should be unlikely on windows), the mentioned `operation forbidden` error is returned.

**Comment by tabramczyk** [comment]
I don't see GUI window. Probably oidc-prompt has problem with showing it.

I've tested it with Administrator permissions on both cmd and PowerShell.

I tried on WSL too and it works when I install `apt install 'oidc-agent-desktop=5.*'` so it looks not problem with my Windows configuration/antivirus. 

Sorry for slow response but I was sick.

**Comment by zachmann** [comment]
Can you try to run `oidc-prompt` manually and see what happens:
```
oidc-prompt password title text label 300
```

**Comment by tabramczyk** [comment]
Nothing showed. 
Im sending output from sysinternals's Process Monitor with 2 formats: CSV and PML. I see some BUFFER OVERFLOW results and some unexpected paths like C:\Program Files\PuTTy there.
[Logfile-oidc-prompt.zip](https://github.com/indigo-dc/oidc-agent/files/13937178/Logfile-oidc-prompt.zip)

**Comment by zachmann** [comment]
Can you verify that the installation directory contains a `oidc-webview.exe`.

The installation dir should contain the following executables:
```
- oidc-add.exe
- oidc-agent.exe
- oidc-gen.exe
- oidc-plugin.exe
- oidc-prompt.exe
- oidc-token.exe
- oidc-webview.exe
```

**Comment by tabramczyk** [comment]
Under `C:\Program Files\oidc-agent\oidc-agent` I have all of this executables.

**Comment by tabramczyk** [comment]
Can I somehow test webview.exe itself? There is no --help there.
Or is there anything else I can provide you too help finding solution?

**Comment by tabramczyk** [comment]
It works for simple HTML below:
`<html>
<body>
<h1>Hello World!</h1>
</body>
<html>`

Effect:
<img width="232" alt="image" src="https://github.com/indigo-dc/oidc-agent/assets/13380347/875e8068-7480-4ca9-9fd7-dc3cf935153c">

**Comment by tabramczyk** [comment]
Any ideas what can be the problem why oidc-token can't show window in this case?

**Comment by tabramczyk** [comment]
I'm not expert in sysinternals tools but it looks like it can't find oidc-webview.exe and starts to search it everywhere:
<img width="351" alt="image" src="https://github.com/indigo-dc/oidc-agent/assets/13380347/6dcd5021-71e4-4ab3-968e-68a7bfe397c7">

As I said before i have oidc-webview exe in standard installation path: C:\Program Files\oidc-agent\oidc-agent\oidc-webview exe.

**Comment by zachmann** [comment]
Additionally, could you try to install the `5.0.2` version (preview) from https://repo.data.kit.edu/prerel/windows/oidc-agent/
and see if that fixed the problem - there was some change related to windows paths.

**Comment by tabramczyk** [comment]
TL;DR;
It's our fault.

Long story:
oidc-agent installer is blocked by Windows Defender so we took it (installer)  to our Security and after their checks we put installer to our internal applications manager that installs it in our machines. Managers broke something within installation process and we ended up without oidc-agent in Windows' PATH environment variable.
After adding oidc-agent to PATH everything works fine in this case.

I've tried install oidc-agent `5.0.1` directly and it works fine.

Sorry for the problem and thanks for help.
